src/main/resources/db/changelog/changes/10-create-cart-items-table.yaml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
+  - changeSet:
+      id: 10-create-cart-items-table
+      author: petro.feniak
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: shopping_cart_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: int
+                  constraints:
+                    nullable: false
+        - addForeignKeyConstraint:
+            baseColumnNames: shopping_cart_id
+            baseTableName: cart_items
+            constraintName: fk_cart_item_shopping_cart
+            referencedColumnNames: id
+            referencedTableName: shopping_carts
+        - addForeignKeyConstraint:
+            baseColumnNames: book_id
+            baseTableName: cart_items
+            constraintName: fk_cart_item_book
+            referencedColumnNames: id
+            referencedTableName: books
```

Comment:
```suggestion
            referencedTableName: books

```
![image](https://github.com/user-attachments/assets/f695b9bf-0b08-41bf-bfae-418bbf121571)
always review your PR, if you see this red sign, it means you need to add one more empty line